ci/cron/daily-compat.yml

```diff
@@ -173,6 +173,94 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
 
+  - job: perf_http_json_oracle
+    timeoutInMinutes: 120
+    pool:
+      name: "ubuntu_20_04"
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          source $(bash_lib)
+
+          docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD"
+          IMAGE=$(cat ci/oracle_image)
+          docker pull $IMAGE
+          # Cleanup stray containers that might still be running from
+          # another build that didn’t get shut down cleanly.
+          docker rm -f oracle || true
+          # Oracle does not like if you connect to it via localhost if it’s running in the container.
+          # Interestingly it works if you use the external IP of the host so the issue is
+          # not the host it is listening on (it claims for that to be 0.0.0.0).
+          # --network host is a cheap escape hatch for this.
+          docker run -d --rm --name oracle --network host -e ORACLE_PWD=$ORACLE_PWD $IMAGE
+          function cleanup() {
+              docker rm -f oracle
+          }
+          trap cleanup EXIT
+          testConnection() {
+              docker exec oracle bash -c 'sqlplus -L '"$ORACLE_USERNAME"'/'"$ORACLE_PWD"'@//localhost:'"$ORACLE_PORT"'/ORCLPDB1 <<< "select * from dba_users;"; exit $?' >/dev/null
+          }
+
+          # dont want to wait forever to test connection , 15m is more than sufficient here.
+          declare -fxr testConnection
+          timeout 15m bash -c 'until testConnection; do echo "Could not connect to Oracle, trying again..." ; sleep 1 ; done'
+
+          # Actually run some tests
+
+          bazel build //ledger-service/http-json-perf/...
+          DAR="${PWD}/bazel-bin/ledger-service/http-json-perf/LargeAcs.dar"
+          JWT="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU"
```

Comment:
Would be nice to put (in a comment, perhaps) what that JWT decodes to, to save people a round trip to jwt.io.